packages/core/.storybook/preview.js

```diff
@@ -72,19 +77,24 @@ export const parameters = {
       ],
     },
   },
-  cssresources: [
-    {
-      id: 'darktheme',
-      code: `
-      <style>
-        /* coming soon dark theme demo (will load external dark theme CSS file) */
-        body { background-color: hsl(201, 30%, 15%); }
-      </style>`,
-      picked: false,
+  themes: {
+    list: [
+      { name: 'light', class: 'cds-theme-light', color: 'hsl(0, 0%, 100%)' },
+      { name: 'dark', class: 'cds-theme-dark', color: 'hsl(211, 63%, 14%)' },
+    ],
+    clearable: false,
+    onChange: theme => {
+      const doc = document.querySelector('#storybook-preview-iframe').contentWindow.document;
+      doc.body.setAttribute('cds-theme', theme.name);
+      doc
+        .querySelectorAll(`[id*='story--'], [cds-theme]`)
+        .forEach(story => story.setAttribute('cds-theme', theme.name));
     },
-  ],
+  },
```

Comment:
This logic allows me to apply the dark theme to specific stories and the story demo frame.